package.json

```diff
@@ -30,6 +31,7 @@
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.16.7",
     "@babel/register": "^7.15.3",
+    "@redocly/cli": "^1.26.1",
```

Comment:
Does this dependency need to be available at production compilation, or can it be moved to `devDependencies` to improve production install times?

Ref: https://github.com/18F/identity-idp/blob/main/docs/frontend.md#dependencies

Reply:
updated in https://github.com/18F/identity-idp/pull/11689/commits/62f42b81f26494a577d16ff2b624ae3f5f80d7bd